src/main/java/org/lobid/resources/run/WikidataGeodata2Es.java

```diff
@@ -176,6 +183,35 @@ public static void loadQidMap() {
 		LOG.info("... loaded " + qidMap.size() + " entries from QID csv.");
 	}
 
+	/**
+	 * Loads the notations from nwbib-skos.
+	 */
+	public static void loadNotationMap() {
+		LOG.info("going to load 'notation' from " + NOTATION_CSV_FN + "...");
+		String line = null;
+		try {
+			lineReader = new BufferedReader(new FileReader(NOTATION_CSV_FN));
```

Comment:
In case you didn't know, there's a new-ish way of reading lines that I like since it's very concise. Basic idea:

```Java
for (String line : Files.readAllLines(Paths.get(NOTATION_CSV_FN))) {
  String[] nwbibSpatialTsv = line.split("\\|");
  //...
}
```

May be useful for the next time or something.